kafkabp/config.go

```diff
@@ -99,6 +100,12 @@ type ConsumerConfig struct {
 	// or it might make things worse.
 	// You are advised to test before using non-empty rack id in production.
 	RackID RackIDFunc `yaml:"rackID"`
+
+	// Optional. Defaults to 5 seconds.
+	//
+	// This affects how often the kafka consumer writes its offsets back to
+	// the cluster.
+	AutoCommitIntervalSeconds int `yaml:"autoCommitInterval"`
```

Comment:
```suggestion
	AutoCommitIntervalSeconds int `yaml:"autoCommitIntervalSeconds"`
```

That said... I would keep this as `time.Duration` 😅 

I can see the argument for doing it this way as a way to enforce not setting a time below one second